ansible/roles/base/tasks/execute.yml

```diff
@@ -0,0 +1,46 @@
+- name: check if socket exists
+  stat:
+    path: /tmp/firecracker.socket
+  register: socket_exists
```

Comment:
you're not doing anything with it. Either remove it, or do something.

Reply:
yup, I forgot to use it

Reply:
Added